src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -20,6 +20,7 @@
       <dependency id="Microsoft.DotNet.TestHost" version="1.0.3-prerelease" />
       <dependency id="Microsoft.DotNet.CoreCLR" version="1.0.0-prerelease" />
       <dependency id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+      <dependency id="NuProj" version="0.10.4-beta-gf7fc34e7d8" />
```

Comment:
I just removed all the BuildTools package dependencies and similarly this shouldn't be a reference package dependency.